test/models/homepage_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+describe Homepage do
+  # it "does a thing" do
+  #   value(1+1).must_equal 2
+  # end
+end
```

Comment:
No tests for homepage?


```suggestion
describe Homepage do
  it "loads properly" do
    get root_path
    
    must_respond_with :success
  end
  
  it "loads with no works" do
    Work.destroy_all
    
    get root_path
    
    must_respond_with :success
  end
end
```